prisma/schema.prisma

```diff
@@ -16,6 +16,11 @@ enum Role {
   TEACHER
 }
 
+enum PrivatePosts {
+  PUBLIC
+  PRIVATE
+}
```

Comment:
I'm thinking that this enum could potentially be re-used in other models if we change the name of it. What if, down the line, we decide we want to allow profile pages to be public / private? I think a better name for this enum would be `Visibility`

Delete both of the migration files you generated, change the name of this then generate your migrations